client/scripts/startup.js

```diff
@@ -1,13 +1,24 @@
 /* eslint-disable global-require */
-function start() {
-  require('../app/bundles/Course/LessonPlan/startup');
+/* eslint-disable import/no-dynamic-require */
+
+function loadCurrentModule() {
+  const modulePath = document.head.querySelector("[name='server-context']")
+                       .getAttribute('data-controller-name');
+
+  try {
+    require(`../app/bundles/${modulePath}`);
```

Comment:
From this, if there are common javascript functions, they should be placed in `/helpers`, and imported if required?